### DISCUSSION TITLE
Making use of GitHub features

### DISCUSSION THREAD

**Discussion by rwp0** [StrawberryPerl/Perl-Dist-Strawberry#112] [Ideas]
Adding website, description, topics (tags) so "About" is not empty.

![image](https://github.com/StrawberryPerl/Perl-Dist-Strawberry/assets/68062695/67c84e49-8891-4e79-ac9e-a09ec9e14aee)

https://strawberryperl.com/

Topics like [`perl`](https://github.com/topics/perl), `windows`, and `cpan` can increase search/discoverability.

**Comment by genio** [comment]
Done. Thanks!!
